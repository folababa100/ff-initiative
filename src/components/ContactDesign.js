import React from 'react';

const ContactDesign = () => {
  return (
    <div className="container">
      <div className="row">
        <div className="col-12">
          <div className="text-center">
            <h3>Welcome to our Contact Page</h3>
          </div>
        </div>
      </div>
    </div>
  )
}

export default ContactDesign;
