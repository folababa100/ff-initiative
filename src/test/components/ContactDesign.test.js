import React from 'react';
import ContactDesign from '../../components/ContactDesign';
import { shallow } from "enzyme";

test('should render component correctly', () => {
  const wrapper = shallow(<ContactDesign/>);
  expect(wrapper).toMatchSnapshot();
})

