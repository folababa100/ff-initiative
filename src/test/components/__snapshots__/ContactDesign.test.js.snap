// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ContactDesign />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="row"
>
        <div
                className="col-12"
        >
                <div
                        className="text-center"
                >
                        <h3>
                                Welcome to our Contact Page
                        </h3>
                </div>
        </div>
</div>,
      "className": "container",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="col-12"
>
          <div
                    className="text-center"
          >
                    <h3>
                              Welcome to our Contact Page
                    </h3>
          </div>
</div>,
        "className": "row",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="text-center"
>
            <h3>
                        Welcome to our Contact Page
            </h3>
</div>,
          "className": "col-12",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h3>
              Welcome to our Contact Page
</h3>,
            "className": "text-center",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Welcome to our Contact Page",
            },
            "ref": null,
            "rendered": "Welcome to our Contact Page",
            "type": "h3",
          },
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="row"
>
          <div
                    className="col-12"
          >
                    <div
                              className="text-center"
                    >
                              <h3>
                                        Welcome to our Contact Page
                              </h3>
                    </div>
          </div>
</div>,
        "className": "container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="col-12"
>
            <div
                        className="text-center"
            >
                        <h3>
                                    Welcome to our Contact Page
                        </h3>
            </div>
</div>,
          "className": "row",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="text-center"
>
              <h3>
                            Welcome to our Contact Page
              </h3>
</div>,
            "className": "col-12",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <h3>
                Welcome to our Contact Page
</h3>,
              "className": "text-center",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Welcome to our Contact Page",
              },
              "ref": null,
              "rendered": "Welcome to our Contact Page",
              "type": "h3",
            },
            "type": "div",
          },
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getSnapshotBeforeUpdate": true,
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
